CppCoreGuidelines.md

```diff
@@ -8379,7 +8379,7 @@ The problem is that the unqualified and qualified lookups both have uses.
 ##### Reason
 
 The `&` operator is fundamental in C++.
-Many parts of the C++ semantics assumes its default meaning.
+Many parts of the C++ semantics assume its default meaning.
```

Comment:
I personally like this, but opinions differ on whether "semantics" should take a singular or plural verb.

Reply:
I don't disagree – just thought that it might read better this way. I can of course revert this change.

Reply:
(Actually, to me it sounds like "parts" is key here, and not "semantics." But I'm not a native speaker, and I [have had trouble with similar matters also before](https://english.stackexchange.com/questions/5378/a-number-of-questions-has-been-or-have-been-asked). :)

Reply:
Oh, you're absolutely right! It should definitely be a plural verb here. I got distracted by "semantics" but that's not actually relevant here.